packages/scalecube-microservice/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "commonjs",
+    "lib": ["dom", "es2015", "es2017.object", "es5", "es6"],
+    "declaration": true,
+    "outDir": "./lib",
+    "strict": true,
+    "noImplicitAny": false,
```

Comment:
suggest to use true here to make it more strict

Reply:
good idea, we don't have legacy code so there is no need to allow it